src/Jackalope/Transport/Logging/DebugStack.php

```diff
@@ -28,6 +28,13 @@ class DebugStack implements LoggerInterface
     public $enabled = true;
 
     /**
+     * Show the debug backtrace for each call
+     *
+     * @var boolean
+     */
+    public $backtrace = false;
```

Comment:
okay. enabled seems never used outside the class either. not sure, but i don't care too much here right now. merged so we can see if the bundle works too.